test/sim/abilities/emergencyexit.js

```diff
@@ -83,15 +83,17 @@ describe(`Emergency Exit`, function () {
 		assert(!battle.p2.activeRequest.forceSwitch);
 	});
 
-	it.skip(`should request switch-out before end-of-turn fainted Pokemon`, function () {
-		battle = common.createBattle([
-			[{species: "Golisopod", ability: 'emergencyexit', item: 'toxicorb', moves: ['substitute', 'sleeptalk', 'liquidation']}, {species: "Magikarp", moves: ['splash']}],
-			[{species: "Charizard", item: 'rockyhelmet', moves: ['bellydrum', 'sleeptalk', 'dragonclaw']}, {species: "Mew", moves: ['sleeptalk']}],
-		]);
-		battle.makeChoices('move substitute', 'move bellydrum');
-		battle.makeChoices('move sleeptalk', 'move sleeptalk');
-		battle.makeChoices('move liquidation', 'move dragonclaw');
+	it(`should request switch-out before end-of-turn fainted Pokemon`, function () {
+		battle = common.createBattle([[
+			{species: "Golisopod", item: 'blacksludge', ability: 'emergencyexit', moves: ['payback']},
```

Comment:
Can you use an OHKO move instead of Payback? It'd make it more obvious the Swoobat is intended to die.

Reply:
tbh I'm not sure how you can make it any more obvious than `assert.fainted`.

Reply:
either way is fine, that's a very minor nitpick